package.json

```diff
@@ -21,7 +21,7 @@
   "types": "lib/index.d.ts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/SwissDataScienceCenter/renku-jupyterlab-ts.git"
+    "url": "git+https://github.com/SwissDataScienceCenter/renku-jupyterlab-ts.git"
```

Comment:
You should probably not include `git+` in the url according to npm's specifics https://docs.npmjs.com/files/package.json#repository

Reply:
oh really? `npm init` actually added it...

Reply:
Well, it probably belongs there after all. I believe the code more than the documentation :smile: